release-notes/2.1/2.1.0-commit.md

```diff
@@ -198,7 +198,7 @@ This is a somewhat curated list of commits across the entire .NET Core 2.1 devel
 * [`[f050264]`](https://github.com/dotnet/cli/commit/f050264) Rename to BundledNETCorePlatformsPackageVersion (#8538)
 * [`[b69d9bd]`](https://github.com/dotnet/cli/commit/b69d9bd) Updating the MSI Installation Note message. Also had to shift components around slightly to make it fit.
 * [`[d3263aa]`](https://github.com/dotnet/cli/commit/d3263aa) Fixing recent CLI test failures around ASP.NETcore in the orchestrated build (#8553)
-* [`[4ac4751]`](https://github.com/dotnet/cli/commit/4ac4751) The BuildTasksFeedTool version and the VersionTools version should be in sync to prevent incompatabilities between these two packages.
+* [`[4ac4751]`](https://github.com/dotnet/cli/commit/4ac4751) The BuildTasksFeedTool version and the VersionTools version should be in sync to prevent incompatibilities between these two packages.
```

Comment:
```suggestion
* [`[4ac4751]`](https://github.com/dotnet/cli/commit/4ac4751) The BuildTasksFeedTool version and the VersionTools version should be in sync to prevent incompatabilities between these two packages.
```